src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -40,7 +39,51 @@ static void Main()
         Console.Write(rn);
     }
 }";
-            CompileAndVerify(source: source, expectedOutput: "TwoZero");
+            CompileAndVerify(source: source, expectedOutput: "TwoZero")
+                .VerifyOperationTree(@"
+Test.Main()
```

Comment:
Yes, please do not merge multiple verifications in one statement.

---
In reply to: [65421246](https://github.com/dotnet/roslyn/pull/11680#discussion_r65421246) [](ancestors = 65421246)